File: PRAK205-2410817310009-Muhammad_Ghazi_Rakhmadi.py
import math

A = float(input("Masukkan panjang sisi A (tinggi): "))
B = float(input("Masukkan panjang sisi B (sisi miring): "))

tinggi = A
sisi_miring = B
alas = math.sqrt(B**2 - A**2)
keliling = alas + tinggi + sisi_miring
luas = 0.5 * alas * tinggi

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

print("\n")

A = float(input("Masukkan panjang sisi A (tinggi): "))
B = float(input("Masukkan panjang sisi B (sisi miring): "))

tinggi = A
sisi_miring = B
alas = math.sqrt(B**2 - A**2)

keliling = alas + tinggi + sisi_miring
luas = 0.5 * alas * tinggi

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: PRAK205-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>
#include <math.h>

int main() {
    float A1, B1, C1, alas1, tinggi1, keliling1, luas1;

    printf("Masukkan panjang sisi A (tinggi): ");
    scanf("%f", &A1);
    printf("Masukkan panjang sisi B (sisi miring): ");
    scanf("%f", &B1);

    tinggi1 = A1;
    C1 = B1;
    alas1 = sqrt(pow(B1, 2) - pow(A1, 2));

    keliling1 = alas1 + tinggi1 + C1;
    luas1 = 0.5 * alas1 * tinggi1;

    printf("Alas = %.0f cm\n", alas1);
    printf("Tinggi = %.0f cm\n", tinggi1);
    printf("Keliling = %.0f cm\n", keliling1);
    printf("Luas = %.0f cm^2\n", luas1);

    printf("\n");

    float A2, B2, C2, alas2, tinggi2, keliling2, luas2;

    printf("Masukkan panjang sisi A (tinggi): ");
    scanf("%f", &A2);
    printf("Masukkan panjang sisi B (sisi miring): ");
    scanf("%f", &B2);

    tinggi2 = A2;
    C2 = B2;

    alas2 = sqrt(pow(B2, 2) - pow(A2, 2));

    keliling2 = alas2 + tinggi2 + C2;
    luas2 = 0.5 * alas2 * tinggi2;

    printf("Alas = %.0f cm\n", alas2);
    printf("Tinggi = %.0f cm\n", tinggi2);
    printf("Keliling = %.0f cm\n", keliling2);
    printf("Luas = %.0f cm^2\n", luas2);

    return 0;
}